projects/1.20/assets/toms-peripherals/toms_peripherals/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+	"block.toms_peripherals.gpu": "图形处理器",
+	"block.toms_peripherals.monitor": "显示器",
+	"block.toms_peripherals.wdt": "超时复位计时器",
+	"block.toms_peripherals.redstone_port": "红石接口",
+	"block.toms_peripherals.keyboard": "键盘",
+	"block.toms_peripherals.keyboard_dongle": "无线键盘信号接收器",
+	
+	"item.toms_peripherals.gpu_chip": "图形处理器核心",
+	"item.toms_peripherals.gpu_chip_raw": "未经烧制的图形处理器核心",
+	"item.toms_peripherals.laser_pointer": "激光笔",
+	"item.toms_peripherals.portable_keyboard": "无线键盘",
+	
+	"itemGroup.toms_peripherals.tab": "Tom's Peripherals(汤姆的外设)",
+	
+	"label.toms_peripherals.wdt.enabled": "启用",
+	"label.toms_peripherals.wdt.disabled": "禁用",
+	"label.toms_peripherals.wdt.timeLimit": "超时: %s",
+	"label.toms_peripherals.wdt.timer": "用时: %s",
+	"label.toms_peripherals.keyboard.ccmon": "左键激活显示器",
+	"label.toms_peripherals.keyboard.monitor": "已启用鼠标支持",
+	"label.toms_peripherals.keyboard.open": "键盘已开，按住ESC关闭",
+	
+	"config.toms_peripherals.maxScreenSize": "屏幕尺寸",
+	"config.toms_peripherals.maxVRAMSize": "显存",
+	
+	"tooltip.toms_peripherals.hold_shift_for_info": "按住 [Shift] 获取更多信息.",
+	"tooltip.toms_peripherals.monitor": "图形处理器必须与显示屏的左下角接触，箭头应指向远离显示器的方向.",
+	"tooltip.toms_peripherals.gpu": "驱动显示器并执行图像操作，有关详细功能列表，请参阅 Wiki.",

```

```suggestion
	"tooltip.toms_peripherals.gpu": "驱动显示器并执行图像操作，有关可用函数列表，请参阅Wiki。",
```
`函数`部分为原文
半角字符与全角字符间无需保留空格，全角句号